openpype/tools/pyblish_pype/settings.py

```diff
@@ -24,4 +24,4 @@
 }
 
 # Allow animations in GUI
-Animated = env_variable_to_bool("OPENPYPE_PYBLISH_ANIMATED", True)
+Animated = env_variable_to_bool("OPENPYPE_PYBLISH_ANIMATED", False)
```

Comment:
I would rather just turn off animations if publishing is in progress instead of turning animations completely by default.

Reply:
That could work nice too! Yes. Would that be trivial?

Reply:
Made PR for that https://github.com/BigRoy/OpenPype/pull/1/files